data/hai/hai reveal 1 intro.txt

```diff
@@ -991,8 +1012,9 @@ mission "Hai Leaks Response 6"
 				`	"Am I allowed to know more?"`
 			`	Alondo gives you a quizzical look. "Of course you are, <last>."`
 			label about
-			`	He pulls out a hand tablet and shows you a title screen which doesn't automatically look familiar. As he talks he flicks through a series of reference images. "There's an older TV show franchise with a cult following called 'Our Fuzzy Paradise', which has had a few different reboots and iterations over the decades. In case you haven't guessed, it features a whole cast of anthropomorphic squirrels. Most recently we had a seven season series run subtitled 'Fellowship is Glamorous' which was one of the more successful ones.`
-			`	"The production of the very first series was entirely funded through a government grants program though, as a cover for any accidental leaks about the Hai. Xilin tells me that since then it's pretty much run itself. The idea here is that since the last one was so successful, someone has decided that it's worth the risk to make a big budget movie production out of it and to potentially use that to kick off a sort of tourist trap type theme park destination. We've found a Seerwood producer who's willing to be all wounded on TV about the spoiling of his 'big surprise' release, and we're passing off the footage as leaked promotional material. For the sake of the cover, we've also decided that this project has a 'fan community' that have been hired on as a focus group to support the production for consistency and accuracy."`
+			`	He pulls out a hand tablet and shows you a title screen which doesn't automatically look familiar. As he talks he flicks through a series of reference images.`
+			`	"There's an older TV show franchise with a cult following called 'Our Fuzzy Paradise' with a whole cast of anthropomorphic squirrels. The production of the first series was entirely funded through a government grants program, as a cover for accidental leaks about the Hai. Xilin tells me that since then it's pretty much run itself.`
+			`	"We're going to make a tourist trap type theme park destination, and advertise it with a big-budget movie production of the show. We've found a Seerwood producer who'll spoil a 'big surprise' release, and we're passing off the footage as leaked promotional material. For the sake of the cover, we've also hired a 'fan community' to support the production for consistency and accuracy."`
```

Comment:
RIP "Fellowship is Glamorous."

Reply:
I put back in the full description, but I let the player interrupt and skip everything after the first paragraph. That's the same way I'm dealing with Turner's movie rambling in my other PRs.  (I hadn't thought of that approach yet when I was reviewing part 1.)